laravel/app/Http/Controllers/ReportController.php

```diff
@@ -4,6 +4,8 @@
 
 use Illuminate\Http\Request;
 
+use App\Helpers;
```

Comment:
I like how you created a helpers folder, this is a convention I've used on other projects! :+1:

Reply:
Right now it's a file technically.

When it gets to be bigger I can make the split, unless you'd like me to go ahead and nip that now.

Reply:
Ahh, good point. I think a file is sufficient for now, if we need more helper files in the future we can update it.